src/Distribution/Client/Add.hs

```diff
@@ -85,12 +86,25 @@ data Config = Config
   , cnfComponent :: !(Either CommonStanza ComponentName)
   -- ^ Which component to update?
   -- Usually constructed by 'resolveComponent'.
-  , cnfDependencies :: !(NonEmpty ByteString)
-  -- ^ Which dependencies to add?
+  , cnfAdditions :: !(NonEmpty ByteString)
+  -- ^ Which content to add to the target field?
   -- Usually constructed by 'validateDependency'.
+  , cnfTargetField :: !TargetField
+  -- ^ Which field to add the provided content to?
```

Comment:
Let's swap the order of `cnfAdditions` and `cnfTargetField`, otherwise reading documentation in order is somewhat confusing.